configuration/src/configs/general.rs

```diff
@@ -95,16 +101,23 @@ impl FromStr for ChainId {
     }
 }
 
-#[derive(Deserialize, Debug, Clone)]
+#[derive(Validate, Deserialize, Debug, Clone)]
 pub struct CommonGeneralRpcServerConfig {
     #[serde(deserialize_with = "deserialize_optional_data_or_env", default)]
     pub server_port: Option<u16>,
     #[serde(deserialize_with = "deserialize_optional_data_or_env", default)]
     pub max_gas_burnt: Option<u64>,
+    #[validate(range(min = 0.0, message = "Contract code cache size must be greater than 0"))]
```

Comment:
Would `min = 0.0` allow that value? I mean the error message clearly says it should be greater than 0 and I suspect that `min` can allow it